auth0/src/main/java/com/auth0/android/provider/OAuthManager.java

```diff
@@ -335,18 +286,6 @@ private boolean shouldUsePKCE() {
         return parameters.containsKey(KEY_RESPONSE_TYPE) && parameters.get(KEY_RESPONSE_TYPE).contains(RESPONSE_TYPE_CODE) && PKCE.isAvailable();
```

Comment:
On the migration notes you mention code+PKCE is the only supported flow. Seems like this method is no longer relevant.

Reply:
Related to how to handle something that should never happen, we could remove it along with any attempt to handle a non-recoverable situation that should never happen. I suppose the question is if we should send the exception back on the error callback instead of just letting the exception propagate.

Reply:
Updated to throw an exception if PKCE is not available when starting login.